tasks/README.md

```diff
@@ -20,20 +20,20 @@ The complete list of tasks can be seen by running `npx hardhat --help` in your t
 | The bare `deploy` task is unrelated to our deployment process and SHOULD NOT BE RUN. It will be disabled/overridden in the future     |
 
 ```sh
-  deploy                Deploy contracts
-  deploy:all            deploys the HyperdriveFactory, all deployer coordinators, and all hyperdrive instances
-  deploy:coordinator    deploys the HyperdriveDeployerCoordinator with the provided name and chain
-  deploy:factory        deploys the HyperdriveFactory with the provided name and chain
-  deploy:instance       deploys the Hyperdrive instance with the provided name and chain
-  deploy:registry       deploys the hyperdrive factory to the configured chain
-  etherscan-verify      submit contract source code to etherscan
+  deploy:hyperdrive     deploys the HyperdriveFactory, all deployer coordinators, and all hyperdrive instances
+  deploy:verify         attempts to verify all deployed contracts for the specified network
 ```
 
-## Configuration
+The `deploy:hyperdrive` task should be run for all deployments. It handles resuming existing deploys
+and only deploys configurations that are not already present on the specified chain. It must be
+provided a `--network` flag.
```

Comment:
I think it would be good to add an examples section that shows the exact command used to deploy and verify. That'll help people when they are getting started so they don't need to figure out the cli args